caravel/assets/spec/javascripts/sqllab/QuerySearch_spec.jsx

```diff
@@ -5,26 +5,60 @@ import QuerySearch from '../../../javascripts/SqlLab/components/QuerySearch';
 import { shallow } from 'enzyme';
 import { describe, it } from 'mocha';
 import { expect } from 'chai';
+import sinon from 'sinon';
 
 describe('QuerySearch', () => {
+  const wrapper = shallow(<QuerySearch />);
   it('should render', () => {
     expect(
       React.isValidElement(<QuerySearch />)
     ).to.equal(true);
   });
 
-  it('should have two Select', () => {
-    const wrapper = shallow(<QuerySearch />);
-    expect(wrapper.find(Select)).to.have.length(2);
+  it('should have four Select', () => {
+    expect(wrapper.find(Select)).to.have.length(4);
+  });
+
+  it('should update userId', () => {
```

Comment:
i generally think using `should` in tests is not neccessary. could make this more clear: `it('updates userId on user select change')`